### ISSUE TITLE
Remove old Meeting Agendas from Wiki

### ISSUE THREAD

**Issue by JackHaeg** [hackforla/VRMS#1636]
### Overview
Currently, we have meeting agendas stored on both the Wiki and GitHub. Old meeting agendas stored on the Wiki need to be moved over to GitHub to centralize our meeting agendas and simplify our Wiki.

### Action Items
- [x] Copy & paste each meeting as a comment on this issue, with the date as the title (format = YYYY-MM-DD)
- [x] Delete old meeting agendas from the VRMS Wiki
- [x] Add a link to this issue to the current Agenda issues on HackforLA: 
  - https://github.com/hackforla/VRMS/issues/1280
  - https://github.com/hackforla/VRMS/issues/1519
- [x] Delete existing Team Meeting page from Wiki 

### Resources/Instructions
Team meetings on Wiki can be accessed here: https://github.com/hackforla/VRMS/wiki/Team-Meetings

**Comment by JackHaeg** [comment]
# 2022-07-11
## Team Meeting Agenda

## Pre-work to prep for meeting

- [ ] Update issues

## Who was there?

- Josh
- Shaun
- Kat
- Julia
- Judy

## Things we plan to discuss
- [ ] Meetings and thoughts on how to run them!
    - [ ] Agendas
    - [ ] Note taker
    - [ ] Time keeper
    - [ ] Facilitator
- [ ] Review [v0.3 project board](https://github.com/hackforla/VRMS/projects/2)
  - [ ] In progress column
  - [ ]  prioritized backlog
    - [ ] Make sure that everyone has an issue
    - [ ] Have a discussion on which issues need to be done before moving to v0.4, and which ones could potentially never be completed

  - [ ] Review Google Drive documents - which ones are current and which ones should be archived?
  - [ ] PRD

## What we actually discussed
- [ ] Meetings
    - [ ] Each team member will take on a role for each meeting: (1)Facilitator, (2) Note taker, or (3) Time Keeper. We can rotate each time.
    - [ ] Assigned to Judy: In addition to uploading new meeting agendas in the wiki each week, the agendas are also accessible via VRMS Google Drive.
    - [ ] Meeting Google doc links will also be cross-posted in wiki meeting agendas for easy referencing.
    - [ ] Create how to run meeting guideline: Julia will start this document and post on wiki. Team members will rotate or owned by Julia?
- [ ] Reviewed [v0.3 project board](https://github.com/hackforla/VRMS/projects/2) - See additional notes from Josh below.
- [ ] Design
    - [ ] Julia and Judy will contact Adam to get more information about users / features.
    - [ ] Josh will attend Thursday's design meeting to go over the Figma files. Anyone else will attend?

## Additional notes
 Things to go over in the dev meeting
  - [ ] Josh's redesign of the project page
  - [ ] Moving things from the prioritized backlog into the icebox
  - [ ] Meeting Google Doc [link](https://docs.google.com/document/d/1PqmDtOtYPSEP-7naF2eJCTB-EQhB4dFCefxKRVnQWEM/edit?usp=sharing)

## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-07-14 
## Design Meeting Agenda

## Before the meeting
- [ ] Prepare any issues you want to discuss 
- [ ] Add issues to agenda for approval
- [x] Delegate roles to team members

## Who was there?
- Julia
- Judy
- Josh
- My
- Kat
- Alex

## Things we plan to discuss
- [ ] 10 min - Greetings and introductions, My Lam’s first meeting
- [ ] 20 min - Get in touch with Adam/Bonnie, Update Alex on PM issues, VRMS Drive and docs
- [ ] 20 min - Figma file, FigJam sesh
- [ ] 10 min - Final thoughts, Action Items

## What we actually discussed
- Welcome My!
- Kaban Board 
  - Discussed how the Kaban board works and where the Kaban how-to document is located
    https://docs.google.com/document/d/11Fe7mNdmPBP5bD_yLJ1C0_I1TmoK47AuHHrdhdDyWCs/edit
  - Create an issue for PM to create a user flow on how to use the Kaban board.
- MVP v0.4 Requirements
   - Bonnie confirms that she wants all features on this document.
   - Alex will meet with Josh to discuss how to narrow down the design scope using the MVP v.04 requirements.
   - Team will go though the document and make comments
   - Team agrees to use MVP v0.4 
- Welcome Alex!
   - Showed Alex where the Figma files are.
   - Alex can prioritize v0.4 features and problems to solve.

## Additional notes

## Upcoming Milestones

## Action Items
- [ ] Create a new GitHub board for v0.4
- [ ] All team members read and comment on the MVP v0.4 Requirement document for the 7/21 Design meeting.
- [ ] Alex will meet with Josh to discuss how to narrow down the design scope using the MVP v.04 requirements.
- [x] Judy will make the Monday 7/18/22 agenda

**Comment by JackHaeg** [comment]
# 2022-07-18
## Team Meeting Agenda

## Pre-work to prep for meeting

- [ ] Update issues

## Who was there?
- Julia
- Judy
- Josh
- Shaun
- My
- Alex

## Things we plan to discuss
- [x] Welcoming Alex and My to VRMS! - 10min
- [x] Go over how to run a team meeting in the wiki, assign responsibilities and agree on the agenda
- [x] Assign team members to the following roles
    - [x] Note taker - Julia
    - [x] Time keeper - Judy
    - [x] Facilitator - Joooosh
- [ ] Design Team
  - [ ] Review Google Drive documents - which ones are current and which ones should be archived and how can we maintain? - 15 min
  - [ ] Alex's big questions - 5min
  - [ ] Julias problem statement - 5min
- [ ] closing and action items - 5min



- [ ] Dev Team: Review [v0.3 project board](https://github.com/hackforla/VRMS/projects/2) - 0min
  - [ ] In progress column
  - [ ]  prioritized backlog
    - [ ] Make sure that everyone has an issue
    - [ ] Have a discussion on which issues need to be done before moving to v0.4, and which ones could potentially never be completed

## What we actually discussed
- We assigned team roles and time frames to the agenda
- We introduced ourselves to the team and explained how we run meetings and keep track of the wiki page
- Josh shared his screen and showed us how we get to the wiki page
- Dev team: talk about issues on Thursday (they stayed after the meeting to talk about how the app is broken down)
- Julia briefly mentioned the needs of HfLa and the Design systems insights and concerns:
    - Retention of volunteers
    - Lack of documentation
    - Offboarding
- We went over the drive documents and focused on: MVP v0.4 requirements. 
- Discussing the scope and goal for v4
    - We are designing for desktop first
    - We are going over users and their needs focusing on general volunteers
- Dev team should look over the dev folders in the drive to decide what’s still relevant
- Design team will define a new problem statement and create a new user flow to focus on for v4


## Additional notes
Next meeting
- demo the current v0.4

Action items
- shaun and josh, look at the broken app
- shaun prep the demo of v0.4
- josh review shaun pr
- josh needs to recruit devs
- finalize problem statement
- develop a new user journey onboarding specific from volunteer point of view


## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-07-21
## Design Meeting Agenda

## Before the meeting
- [ ] Team members should update their issues

## Who was there?
- Judy
- Josh
- My
- Julia
- Alex
- Bonnie
- Shaun
- Kat

## Things we plan to discuss
- [ ] Design Systems research findings, how VRMS goals align with HfLA needs
- [ ] Adam Spar meeting
- [ ] VRMS v4 goals: HfLA onboarding
- [ ] New KPIs and success metrics for v4
- [ ] Google drive clean out
- [ ] VRMS new team members onboarding and overall organization for the current team


## What we actually discussed
- [x] Design System
 - HFLA is implementing a design system that sets the ground rules for every team to follow.
 - The goal is to make an easy off-boarding process so information will retain continuity.
 - 4 core issues:
    1. Documentation / organizational memory
    2. Messiness in skills levels, team components, and knowledge.
    3. Knowledge gap among different roles
    4. Lack of internal resources.
- [x] Adam Spar Meeting
 - Ask Adam how to get editor access to Figma.
 - What is the next step for V0.4.
 - Julia and Alex will sit at the next volunteer onboarding meeting.
 - The onboarding process has been complicated and frustrating.
- [x] Bonnie joined the Design meeting
 - Concerned that VRMS will be so complicated to use that she'd rather go back to the old way of managing onboarding.
 - Will join the Adam Spar meeting on Friday 7/22/22.
 - Reminded the Design team that every screen should use components.
 - Bonnie is the only stakeholder that matters for VRMS.
 - The app is not running correctly because volunteers should know which project to go to; it should replace Bonnie during onboarding.
 - There's a document that shows all needed features when Bonnie and Adam met for 2 months to discuss.
 - Figma files:
   - VRMS v1.0 mood board is very important as it explain the need for VRMS to be very plain and simple (stripped down to its basic components.
   - Folder "VRMS Mock - AS/BW New Screen" is the most comprehensive page that Bonnie needs.
 - GitHub:
   - PM (Alex) should make sure that there are enough issues.
   - The Kanban board needs to be cleaned up so she can recruit new developers.
   - Dev Team: clean up V0.3 and V0.4 boards and have actionable issues.
   - Each team will prioritize their own issues.


## Additional notes
- 

## Upcoming Milestones

## Action Items
- [ ] Schedule Zoom meeting with Adam Spar.
- [ ] Alex and Julia will clean up the Kanban board.

**Comment by JackHaeg** [comment]
# 2022-07-25
## Team Meeting Agenda

## Pre-work to prep for meeting

- [ ] Update issues

## Who was there?


## Things we plan to discuss
- [ ] Assign team members to the following roles
    - [ ] Note taker - 
    - [ ] Time keeper -
    - [ ] Facilitator - 

- [ ] PM (10 mins)
   - [ ] Google Drive cleanup status check
   - [ ] Sitting in on next onboard meeting (Tuesday Aug 9, 6pm PT)
   - [ ] Goals: decrease 2.5 hour Avg. onboarding time. Bonnie update (Improve the onboarding form: no verification/validation process).
   - [ ] Fang (from people depot team) joining next Monday July 27th meeting.

[Onboarding form.pdf](https://github.com/hackforla/VRMS/files/9174737/Onboarding.form.pdf)
[Global onboarding form.pdf](https://github.com/hackforla/VRMS/files/9174738/Global.onboarding.form.pdf)

- [ ] Design Team
  - [ ] Problem Statement
  - [ ] Onboarding User Journey Flow
  - [ ] Bonnie wants functionality first --> mobile
  
- [ ] Dev Team
  - [ ] Dev recruitment updates
  - [ ] Demo V4
  - [ ] Dev Team: Review [v0.3 project board](https://github.com/hackforla/VRMS/projects/2)


## What we actually discussed
Brian + Charity: Google Drive Migration. Bonnie showed us how to migrate google shared drive.

Links -
https://drive.google.com/drive/u/0/folders/1yiIBqWFfBJeUOCb42XjJp_SuVc2HOsYJ
https://docs.google.com/spreadsheets/d/1OriRsMHuD26UIZQ5bEsnikqWkLVfTVbQVKZi5RC-8TA/edit#gid=2103419749

Question to be answered:

a) Port V3 to V4 and rewrite. Identifying features that need to be kept for V3 to V4 (Edit projects, edit meeting time, add people to roster and search people, and different permission level, Volunteer check-in)
Pros: Will be able to fix bugs and start clean
Cons: Will take longer to add on new features that would help onboarding


vs.

Add new features in existing V3.
-Pros: Can start claiming small wins (I.e. improving the forms to cut down on onboarding time)
-Cons: May have long term consequences, in terms of bugs and messy code.

Option 3: Would we able to prioritize working on v4 onboarding features and claiming small wins and then port over V3 to V4 after?

To be discussed at next dev meeting.



## Additional notes
Next meeting
- [ ] Dev Team
  - [ ] Dev recruitment updates
  - [ ] Demo V4
  - [ ] Dev Team: Review [v0.3 project board](https://github.com/hackforla/VRMS/projects/2)



## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-07-28
## Design Meeting Agenda

## Before the meeting
- [ ] Team members should update their issues

## Who was there?
- Julia
- Judy
- My
- Alex

## Things we plan to discuss
- [ ] What was discussed on Monday, questions - 20 mins
- [ ] New volunteer onboarding journey (August 9th @ 6:30p pst) - 10 mins
- [ ] Bonnie's pain points (video we recorded with Adam/Bonnie + Monday's meeting notes) - 20 mins
- [ ] Action closing - 5 mins

## What we actually discussed
- [x] What was discussed on Monday
 - Meeting with Bonnie, Julia, Alex, and Adam to talk about the Figma files.
 - VRMS is a tool that will replace Bonnie and a content management system (CMS) tool.
 - V0.4 focus is onboarding.
 - The biggest issue is that V0.3 coding is very messy and buggy. Dev wants to rewrite the V0.4 but build features on top of V0.3.
 - Dev needs to recruit more developers.
 - Dev team should focus on 1 - 2 small wins from V0.3 to help Bonnie help onboarding faster.
 - Tanya is the owner of the Figma file. Bonnie sent an email asking for access.
- [x] New volunteer onboarding
 - Bonnie wants the Dev team to rebuild V0.4.
 - She can use the mobile app since there are screens designed because she has waited long enough.
 - Design will start designing the screens while the Dev team is working on the bugs.

## Additional notes

## Upcoming Milestones
- [ ] Organize Figma files at the next Design meeting.

## Action Items
- [ ] Get access to Figma files to clean up the screens.
- [ ] Julia will send a reminder email to Tanya.
- [ ] Everyone can watch the Zoom meeting recording on their own time.
- [ ] Invite Bonnie to the next Design meeting.

**Comment by JackHaeg** [comment]
# 2022-08-01
## Team Meeting Agenda

## Pre-work to prep for meeting

- [ ] Update issues

## Who was there?


## Things we plan to discuss
- [ ] Assign team members to the following roles
    - [ ] Note taker - Julia
    - [ ] Time keeper - Josh
    - [ ] Facilitator - Alex

- [ ] PM (10 mins)
   - [ ] Google Migration Status Update
   - [ ] Bonnie update: which features are priority
   - [ ] Dev team request: Needs user stories and user journeys for V3 features prioritized

Priority: 

    1. Project level admins need to edit and create new meeting times
    2. Bonnie needs to be able to edit user access levels to different projects
    3. Bonnie needs to be able to add new project
    4. Volunteer check-in
   


Non-Priority:

    1. Delete/edit project information
    2. events page
    3. stats page
    4. Add people to roster and search people

- dev.vrms.io: share email address to get access.

- [ ] Dev Team (10 mins)
  - [ ] Dev recruitment updates
  - [ ] Demo v4
  
- [ ] Design Team (10 mins)
  - [ ] Figma Permission update
  



## What we actually discussed

- We're trying to prioritize which features Bonnie needs to move from v3 to v4 (onboarding)
- Alex send an email to Bonnie to ask about features, we’re waiting for response
- Shaun will screenshot what v3 features we need to pour over
- Dev team will look over the features that are priority and not and confirm them
- Shaun is sharing his screen, v4:
    - Create an account, enter email and password
    - Sign up, get confirmation code from email
    - Add first and last name
    - Homepage
- He will share screenshots of the components that exist for v4
- The dev team has two open roles for developers in the SWE CoP
    - We need to make sure we have clear issues for us to recruit devs
    - We need to clean out the google drive files
- Tanya did not get back to us about transferring figma ownership to VRMS
    - We will probably need to go over the pages, decide which ones are relevant and duplicate them ourselves
    - The design team will meet 08/02 at 7pm to tackle this
- We need to sign into the VRMS Figma team project (with vrms email) to be able to duplicate files
- Alex and Judy got access of v3 VRMS app


## Additional notes
Next meeting:
-Workshop new v0.4 Kanban board on priority features:

    1. Project level admins need to edit and create new meeting times
    2. Bonnie needs to be able to edit user access levels to different projects
    3. Volunteer check-in


## Action Items

- [ ] Design team will go over figma files and duplicate the important ones
- [ ] Dev team will write specific issues to start recruiting
- [ ] Dev team look over priority features and confirm them



## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-08-08
## Team Meeting Agenda

## Pre-work to prep for meeting

- [ ] Update issues

## Who was there?


## Things we plan to discuss
- [ ] Assign team members to the following roles
    - [ ] Note taker - Judy
    - [ ] Time keeper - Julia
    - [ ] Facilitator - Alex

- [ ] Dev Team (10 mins)
    - [x] Walk through screenshots
  
- [ ] Design Team (15 mins)
  - [x] Figma Page duplication update
  - [x] Google drive & file migration

- [ ] PM (20 mins)
   - [ ] Workshop new v0.4 Kanban board on priority features:

    1. Project level admins need to edit and create new meeting times
    2. Bonnie needs to be able to edit user access levels to different projects
    3. Volunteer check-in
  



## What we actually discussed
- [X] Dev Team (10 mins)
  * Shaun walked through screenshots of V4 and V3 old features
  * V.3 doesn’t have the ability to let Bonnie add users to projects.
  * Dev team will write the action plans for the user stories in the Kanban board.
- [X] Design Team (15 mins)
  * Julia went over google drive files and reorganization
  * Julia went over Figma file updates - We duplicated Figma files and transferred ownership to HforLA account
  * Julia shared the updated Figma files and created a VRMS v.4 (new) file
  * Julia defined the users in the Figma files based on Adam’s comments
  * Alex will confirm with the Design team the features. He will create the issues.
- [X] PM (20 mins)
  * Alex went over Kanban board and went over work flow.
  * Alex and Julia will attend and observe the 8/9/22 onboarding meeting.



## Additional notes
With Fang-
- Screenshots of the userflow + Figma



## Action Items




## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-08-11
## Design Meeting Agenda

## Before the meeting
- [ ] Team members should update their issues

## Who was there?
- My
- Julia 

## Things we plan to discuss
- [ ] Assign team members to the following roles
    - [ ] Note taker -
    - [ ] Time keeper - 
    - [ ] Facilitator - Julia

- [ ] New Figma file and organization system

- [ ] Onboarding flow and priority issues
    - [ ] for v3
    - [ ] for v4

## What we actually discussed
- Julia showed My the Figma file and explained organization system
- We talked about the onboarding flow, and what happened during the session on Tuesday
- We talked about the priority v3 features we need to pour over to v4



## Additional notes
- Dev team showed up and we discussed designing screens for v3 to v4 features
     - Design team will come up with user flows for each user story and start designing 

## Upcoming Milestones

## Action Items
- [ ] Design team will come up with user flows for each user story
- [ ] Design team will create sitemap of v4 
- [ ] Design team will start designing screens for the priority features

**Comment by JackHaeg** [comment]
# 2022-08-15
## Team Meeting Agenda

## Pre-work to prep for meeting


## Who was there?


## Things we plan to discuss
- [ ] Dev team requested breakout session to discuss database structure migration from v3 to v4
- [ ] Design team to meet to discuss design sketches.



## What we actually discussed
* Discussed Trillium participation
* Can we record meetings for Trillium? Everyone voted yes.
* Use Kanban board to assign tickets asynchronously

Dev Team Breakout session
- [ ] 

Design Team Breakout session
* Julia to create a style guide for the design team to follow
* Julia over user story flows with team 
* User story 1: Julia
* User story 2: Judy
* USer story 3: My
* User story 4: Julia




## Additional notes



## Action Items




## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-08-18
## Design Meeting Agenda

## Before the meeting
- [ ] Team members should update their issues

## Who was there?
- 

## Things we plan to discuss
- [ ] Going over 'designs for approval'
- [ ] Continue to build style guide
- [ ] Issues or pain points we need to address with devs

## What we actually discussed
- 

## Additional notes
- 

## Upcoming Milestones

## Action Items
- [ ]

**Comment by JackHaeg** [comment]
# 2022-08-22
## Team Meeting Agenda

## Who was there?


## Things we plan to discuss
- [x] Assign team members to the following roles
    - [x] Note taker - My
    - [x] Time keeper - Judy
    - [x] Facilitator - Alex

- [x] Dev Team (5 mins)
    - [x] Database structure updates
  
- [x] Design Team (40 mins)
  - [x] Go over lofi user flow for story #2 - Judy (10 mins)
  - [x] Go over lofi user flow for story #3 - My (10 mins)
  - [x] Discussion (20 mins)

- [x] PM (5 mins)
   - [x] Design workshop date: September 12th.
   - [x] Dev workshop date: September 26th.  

*** Reference
1. Story #1: As a Project Level Admin, I need to be able to create and edit meeting times, so that I can keep HackforLA events page up-to-date.
2. Story #2: As Bonnie, I need to be able to edit user access levels to different projects, so that different volunteers have different admin permissions.
3. Story #3: As Bonnie, I need to be able to add a new project, so that I can manage all the activities going on within HforLA.
4. Story #4: As a volunteer, I need to be able to check-in using VRMS, so that HforLA can keep track of my involvement and attendance.
  



## What we actually discussed
  - [ ] Josh is going to have a meeting on Thursday to discuss how to hold all the project's information in the database.
  - [ ] Judy and My shared wireframes of user flow 2 and 3
  - [ ] Discussed how Bonnie should be able to edit project-related details on multiple screens (e.g assigning volunteers and access level, etc.)

**Notes:
-Bonnie needs to edit time, add project, edit user access levels
-My brought up a good point: We should think about what onboarding would look like and the information to be collected because that will affect the design for the current user stories (#1-4).
-Point of discussion: Who will be assigning volunteers to projects? Would this fall under Bonnie's task as she has user access level editing authority? Or would this fall under the Project Level Admin?

## Additional notes




## Action Items




## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-08-29
## Team Meeting Agenda

## Who was there?


## Things we plan to discuss
- [ ] Assign team members to the following roles
    - [x] Note taker - Julia
    - [x] Time keeper - Kat
    - [x] Facilitator - Alex

- [ ] Intros (15 mins)
    - [ ] Trillium
    - [ ] Tim
  
- [ ] Design Team (10 mins)
  - [ ] Design updates
  - [ ] Question for Dev Team? Is Bonnie using V3 currently?

- [ ] PM (25 mins)
   - [ ] Let's talk metrics: What are we trying to improve? (WAU, # of clicks?, avg. onboarding time, etc) (10 mins)
   - [ ] Reminder about Design workshop date: September 19th (Pushed back from Sept. 12th)
   - [ ] Introduce Team Alignment Deck - Due September 12th (10 mins)
   - [ ] Discuss September 5th meeting availability (Labor Day) (5 mins)

  


## What we actually discussed

- Josh section was moved to another time, he left to talk to Bonnie for a while, instead we’re doing intros.
    - Trillium’s a bootcamp grad and self taught developer and it’s his first time meeting with VRMS.
- Design team went over their bit
    - We have questions for devs and we’ll use the github and kanban board to communicate them
    - We have a Figma file with user flows we want to share
        - Figma filed was shared with all the devs so they can follow along the progress in design and access the deliverables they need
- Alex is filling Trillium and Tim in with the history of VRMS v3 and v4
    - Tim is working on a Slack app that is similar to VRMS feature wise
    - We are continuing to work on VRMS with our current technologies but we would love Tim’s help and input during production, while he continues working on his project
- Alex went over his slide deck. The purpose is for us to each fill in the cards with our names and to get to know each other and our work styles, etc.
- Next Monday, Sept 5h is labor day! WE ARE NOT MEETING. We will resume our meetings the following Thursday Sept 8th
- Design team is going to meet on Wednesday at 8pm instead of Thursday and work on finishing IA docs

## Additional notes




## Action Items
- [ ] Fill out Team Alignment Deck by Sept. 12th




## Upcoming Milestones

# Agenda Backup
## Things we plan to discuss
- [ ] Assign team members to the following roles
    - [ ] Note taker - My
    - [ ] Time keeper - Judy
    - [ ] Facilitator - Alex

- [x] Dev Team (15 mins)
    - [ ] JB's Teachable Moments (10 mins)
    - [ ] Updates (5 mins)
  
- [ ] Design Team (15 mins)
  - [ ] Updates 

- [x] PM (26 mins)
   - [ ] Point of discussion: Who will be assigning volunteers to projects? Would this fall under Bonnie's task as she has user access level  editing authority? Or would this fall under the Project Level Admin? (10 mins)
   - [ ] Reminder: Design workshop date on September 12th. (1 min)
   - [ ] Introduce Team Alignment Deck -- Try to fill by Sept. 5th. (15 min) 


## What we actually discussed
  - [ ]



## Additional notes




## Action Items
- [ ] Fill in Team Alignment Deck 




## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-09-12
## Team Meeting Agenda

## Who was there?

## Things we plan to discuss
- [ ] Assign team members to the following roles (5 mins)
    - [ ] Note taker - 
    - [ ] Time keeper - 
    - [x] Facilitator - Alex

- [ ] PM (25 mins)
   - [ ] Welcome back!: Clarification of VRMS status and where we're headed. Goal setting and timeline. (5 mins)
   - [ ] Going forward: **Bias for action** (with appropriate documentation), 2-3 week sprints, stand-up meetings, and sprint-planning sessions on Mondays. (7 mins)
   - [ ] New PRD for this v0.35 (https://docs.google.com/document/d/1Xgk77wTlrqShCSoJZg0ZCucR-xmoGSE4sMHNRb5Zz4s/edit) (1 mins)
   - [ ] Discuss metrics: What are we trying to improve? (WAU,DAU, avg. onboarding time, etc) (10 mins)
   - [ ] Reminder: Team Alignment Deck - Complete by Sept. 19th so we can discuss (2 mins)

- [ ] Dev (10 mins)
   - [ ] Dev updates: Summary of current status with backend and People Depot (10 mins)

- [ ] Design Team (10 mins)
  - [ ] Design updates
  - [ ] **Set date ** for 1st design hand-offs


   

  


## What we actually discussed
PM

- We want to make sure we prioritize our goals, bias for action we want documentation to let the next cohort know what’s going on when they join. We’ll have 2/3 week sprints and every other Monday we will have a planning meeting to establish what we work on in the next sprint.
- New PRD for v.035 to make sure the team understands what is expected for this version.
    - We are focusing on a few user stories (see github kanban board) and features first before moving onto v.04 Onboarding.
- Let’s thing about metrics and measures about the impact we want to make with VRMS and our time in the team/project. Some metrics:
    - Time it takes a user to complete a task
    - Increase of active users
    - Number of screens and tabs used
- TDM calculator shares VRMS link everytime they have meetings but we don’t know if they actually use them
- Fill out the team alignment deck by next week, MONDAY SEPT 19! please :)

DEV

- People Depot building out the back end so designers can start working on lo/mid-fi wireframes to handoff to devs so they can start creating issues.
- We’re talking about Tim being Dev lead and integrating or work on the Slack app he build. Tim and Josh will continue talking about it with the dev team and Bonnie.

DESIGN

- The design team will meet on WEDNESDAY, SEPT 14 instead of Thursday.
- We will work on User Story 4 which is the volunteer check in first in the next meeting. We’ll work collaboratively to come to an agreement on general layout and look.


## Additional notes




## Action Items




## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-09-14
## Design Meeting Agenda

## Before the meeting
- [ ] Team members should update their issues

## Who was there?
- Julia
- My
- Judy
- Alex
- Stephanie

## Things we plan to discuss
- [X] Catch up with members, project status update (10)
    - [X] IA docs
    - [X] Onboarding flow
    - [X] Team Library and UI kit
    - [X] Other Q's
- [X] Develop User Stories (15-20)
- [ ] Talk about general layout and look, Dashboard (15-20)
- [X] Closing statements, action items (10-20)

## What we actually discussed
- Use the v4.0 UI Requirement Google Doc to start the design.
- Use Stark as Figma plug-in for accessibility.
- Julia is updating the user flows.
- Verifying if we are focusing on User Story 4.
- My asked "what is the purpose of the check-in button?" This will help Bonnie keep track of who attendance, retention, and give metrics to the organization’s impact. You don’t get the Zoom link until they check in.
- Alex raised the question that we need a start / login screen before the dashboard.
- We should come up with a way to encourage users to login in versus punishing them by making it hard for them to get the Zoom link.
- Bonnie wants volunteers to attend the onboarding meeting and then present VRMS after. She wants people to know the time commitment.
- We need to figure out how to do the sign up process. Suggest to separate the user flow to sign up and check in so the dev team can work on the check in flow first.
- We need to know if Bonnie wants to share the link during onboarding.
- Focusing the check in button for the current volunteer.


## Additional notes
- 

## Upcoming Milestones

## Action Items
- [X] Thursday 9/22 Design meeting will work on the wireframes for User Story 4 and 5. 
- [X] Monday 9/26 Team meeting will have something to present.

**Comment by JackHaeg** [comment]
# 2022-09-19
## Design Meeting Agenda

## Who was there?

## Things we plan to discuss
- [ ] Assign team members to the following roles (5 mins)
    - [x] Note taker - Josh
    - [x] Time keeper - Amanda
    - [x] Facilitator - Alex

- [ ] Dev (5 mins)
   - [ ] Dev updates: Tim

- [ ] Design Team (5 mins)
  - [ ] Design updates
  - [ ] Date for handoff: 9/26/22

- [ ] PM (30 mins)
   - [ ] New Team Members
   - [ ] Team Alignment Discussion 

## What we actually discussed


DEV


DESIGN



## Additional notes




## Action Items




## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-09-22
## Design Meeting Agenda
## Before the meeting
- [ ] Team members should update their issues

## Who was there?
- 

## Things we plan to discuss
- [ ] Discuss flows for US 4 and 5, brainstorm wireframes
- [ ] Discuss team library
- [ ] Dashboard

## What we actually discussed
- 

## Additional notes
- 

## Upcoming Milestones

## Action Items
- [ ]

**Comment by JackHaeg** [comment]
# 2022-09-26
## Team Meeting Agenda
## Who was there?

## Things we plan to discuss
- [ ] Assign team members to the following roles (5 mins)
    - [ ] Note taker - Amanda
    - [ ] Time keeper - Timekeeper
    - [ ] Facilitator - Alex

- [ ] Dev (5 mins)
   - [ ] Dev Updates

- [ ] Design Team (30 mins)
   - [ ] Design Updates
   - [ ] User story 5 wireframe
   - [ ] Q&A


- [ ] PM (10 mins)
   - [ ] Dev issues
   - [ ] Design for user stories
   - [ ] New PRD

## What we actually discussed


DEV


DESIGN



## Additional notes
VRMS site walk through with Bonnie

Josh ( Dev ) went over the check in/admin portal
- Task 1: Add new project
- Task 2: Assign a user to a project
- Task 3: Remove user from a project
- Task 4: Edit project meeting time

Bonnie's Feedback
       Task 1:
- Add set locations to select from in the drop down
- No validations in (GITHIB identifier (ID associated with primary repo)
- GITHUB URL ( Name of org and primary repo)
- Slack URL (https://hackforla.slack.com/archives/CRGH5HM0Q )
- Google drive (Url of their shared drive)
- Google drive url ( Url of their shared drive needs validation )
- HFLA website URL ( Validate )
- Video conference link ( remove for the projects) – looking for description (remove this)

        Task 2:
- Assign a user to a project – Home – Admin – User management – Look up User – ( Not sure what to select after ) – Click on name – ( need to update with an edit icon ) 
-	Task 3: Remove a user from project – Admin – User management – remove
Feedback
-       Missing user icons 

-	Task 4: Edit project meeting time – We can only edit events that already exist 
Feedback
-       Need to fix/update this feature
-	Easy to navigate, easy fixes, text seemed small/hard to read.
-	Judy comments: A lot of fixes, focus on fixing one task at a name, UX perspective- *add project has to be specific* for the user and less frustrating, etc.
-	Alex and Judy: Make an issue in GitHub
-	Go to Issues – New issue – Blank issue - *Bold and underline current location for User in Menu* - Add a comment under overview, add user POV and experience (the more info the better, add screenshots) - Add Action items – New issue Approval – PMs will prioritize on the back log (make sure the role and size is on there), Sprint planning – Assign, Move to in-progress, etc.

-	Next team meeting – Have a few issues, as a team go through them and plan for the next sprint
-	Judy- go through what we need to fix from a UX perspective based on Bonnies comments
-	Go through metrics / create measures of improvement and reiterate




## Action Items




## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-10-03
## Team Meeting Agenda
## Who was there?

## Things we plan to discuss
- [ ] Assign team members to the following roles (5 mins)
    - [ ] Note taker - Stephanie
    - [ ] Time keeper - Alex
    - [ ] Facilitator - Amanda

- [ ] Bonnie V3 Run through
   - [ ] 

- [ ] Dev Team updates
   - [ ] Design


- [ ] PM (10 mins)
   - [ ] Updates

## What we actually discussed


DEV


DESIGN



## Additional notes




## Action Items




## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-10-17
## Team Meeting Agenda
What We Plan To Discuss:

* Assign Roles / Recap of Previous Meeting (5 min)
* V3 Priorities (high-level priorities by team based on the discussion w/Bonnie) PM/UX/Dev (15 min)
* Prioritization Criteria - how we determine top priorities (5-10 min)
* Survey to create metrics & measure progress (Judy - 15-20 min)
* Next Steps / Action Items (15 min)

## What we actually discussed - Notes here:
Team went over Bonnie's feedback from the VRMS website walkthrough. 

UI/UX team is looking to prioritize Usability and functionality. 
-Validate certain fields in the project form
-Work on user flow
-Events tab ( Too much confusion ) - Very difficult to change a meeting, needs a updated flow + search ability 
-Consistent design throughout the site
-User friendly design functions
-Update product pages, features, and capabilities

Julia created a Survey and wants to use it a tool for feedback from Bonnie. 
-How do we measure improvements? ( create a rating )
-Team is working on refining questions
-Narrowing down questions
-Divide questions by the 4 key tasks given by Bonnie
- Get written feedback from bonnie to prioritize'
- Work through CSS issues

Trillium and Josh ( Dev Team ) want to work on a function that allows PMs to share/Invite through the App/Website
-Project level admin access
-Focusing on V3 features
-Edit permissions for admins
-Ability to add and remove projects
-Trillium is work on the coding (site bugs)
- Before & after can be found on Dev.VRMS.io

Stephanie/Alex/Amanda ( PM Team )
-Focused on prioritization 
- Focused on getting the team grounded and consistent on what everyone is working on ( Based on the priorities )
- Clean up the board in GitHub , Keep the board updated and relevant to where we are in the project as a team , organize the information.
- Find out what Bonnie needs to use the app 

Next Meetings:

Thursday - Watch video, discuss issues with design team, jot down the errors and rank them. 
Create a go forward plan for design and developers.

Monday - Work through results. And identify what else is preventing bonnie from using the site.

**Comment by JackHaeg** [comment]
# 2022-10-24
## Team Meeting Agenda

## Who was there?

## Things we plan to discuss
- [ ] Assign team members to the following roles (5 mins)
    - [ ] Note taker 
    - [ ] Time keeper 
    - [ ] Facilitator - Amanda

- [ ] Prioritization Items
   - [ ] Review as a team

- [ ] Prioritization Method
   - [ ] Review as a team


- [ ] Prioritization Poker
   - [ ] Team activity

- [ ] Summary for this week
   

## What we actually discussed



## Additional notes




## Action Items




## Upcoming Milestones

**Comment by JackHaeg** [comment]
# 2022-11-03
## Design Meeting Agenda
## Before the meeting
- [ ] Team members should update their issues

## Who was there?
- 

## Things we plan to discuss
- [ ] Next steps for VRMS, prioritize UX issues
    - [ ] Usability Testing
    - [ ] UI Kit
    - [ ] VRMS branding
- [ ] Roadmap for last few meetings of the year
    - [ ] Prioritize issues
    - [ ] Write documentation
    - [ ] Organize ourselves to pick up where we left off
- [ ] Issue created to update VRMS project page on Hfla site

## What we actually discussed
- 

## Additional notes
- 

## Upcoming Milestones

## Action Items
- [ ]

**Comment by JackHaeg** [comment]
Completed 👍
